packages/@react-spectrum/dialog/test/DialogTrigger.test.js

```diff
@@ -136,6 +136,84 @@ describe('DialogTrigger', function () {
     expect(popover).toHaveAttribute('style');
   });
 
+  describe('popover arrows', function () {
```

Comment:
These tests work, but do we still want them if Chromatic covers these cases? The concern is the mocking of the data and test dom elements not really having dimensions.